'use strict';

async function test() {
    try {
    }
    catch (err) {
        console.error(err);
    }
    console.log("\n\nテストが終了しました\n");
}

test();